### PR TITLE
Enhance readme

### PR DESCRIPTION
**Pull request by jasonhe258**
**What this PR does**:
The PR is going to Enhance Readme of the repo.

**Which issue(s) this PR fixes**:

Fixes #

**Type of changes**
Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update

**Feature/Issue validation/testing**:

- Logs

**Special notes for your reviewer**:

**Checklist**:

- [ ] I have added unit/e2e tests that prove your fix is effective or that this feature works.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] I have reviewed my own code and ensured that it follows the project's style guidelines.

**Release note**:

```release-note

```

### CODE REVIEW

**Comment by jasonhe258** [comment]
@Rader @ericyd80 Ready for reviewing. Get the changes view from here: https://github.com/jasonhe258/csghub/tree/enhance_readme Thanks.

File: README.md [hunk 1]
```diff
@@ -1,106 +1,40 @@
...
+
+[![Join WeChat Slack](https://img.shields.io/badge/wechat-join_chat-white.svg?logo=wechat&style=social)](/docs/images/slack-qrcode.png)
+[![Join OpenCSG Slack](https://img.shields.io/badge/slack-join_chat-white.svg?logo=slack&style=social)](https://join.slack.com/t/opencsghq/shared_invite/zt-2fmtem7hs-s_RmMeoOIoF1qzslql2q~A)
+[![Join OpenCSG Discord](https://img.shields.io/badge/discord-join_chat-white.svg?logo=discord&style=social)](https://discord.gg/bXnu4C9BkR)
+
+**[简体中文](/docs/readme_cn.md) ∙ [日本語](/docs/readme_ja.md) ∙ [한국어](/docs/readme_kr.md)**
+
+## CSGHub
+
+CSGHub is an open-source platform designed for managing Large Language Models (LLM) assets, offering an efficient way to handle the entire LLM and their assets such as datasets, spaces and codes etc. It allows users to upload, download, store, verify, and distribute LLM assets via a web interface, `git` command line, or a natural language Chatbot.
```
Comment (Rader) [inline README.md:10]: access CSGHub through sdk is also supported.
add a link to csghub-sdk project

File: README.md [hunk 2]
```diff
@@ -1,106 +1,40 @@
...
-4. join our slack workspace:[OpenCSG Slack Channel](https://join.slack.com/t/opencsghq/shared_invite/zt-2fmtem7hs-s_RmMeoOIoF1qzslql2q~A)
-<div style="display:inline-block">
-<img src="/docs/images/wechat-assistant-new.png" width='200'>
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
-<img src="/docs/images/discord-qrcode.png" width='200'>
-  &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
-<img src="/docs/images/slack-qrcode.png" width='200'>
-</div>
+
+[![Join WeChat Slack](https://img.shields.io/badge/wechat-join_chat-white.svg?logo=wechat&style=social)](/docs/images/slack-qrcode.png)
```
Comment (Rader) [inline README.md:2]: the pic should be `wechat-assistant-new.png`

Comment (jasonhe258) [reply README.md:2]: Great catch!

File: README.md (hunk 1 above)
Comment (jasonhe258) [reply README.md]: Updated @Rader

**Review by Rader — APPROVED** [review]

**Pull request merged**